compiler/src/yul/runtime/functions/mod.rs

```diff
@@ -36,17 +36,6 @@ pub fn std() -> Vec<yul::Statement> {
         abi::pack(AbiDecodeLocation::Memory),
         contracts::create2(),
         contracts::create(),
-        math::checked_add_u256(),
-        math::checked_add_u128(),
-        math::checked_add_u64(),
-        math::checked_add_u32(),
-        math::checked_add_u16(),
-        math::checked_add_u8(),
-        math::checked_add_i256(),
-        math::checked_add_i128(),
-        math::checked_add_i64(),
-        math::checked_add_i32(),
-        math::checked_add_i16(),
-        math::checked_add_i8(),
-    ]
+    ];
+    [fns, math::all()].concat()
```

Comment:
Now that we have alot more functions, we should probably organize all of the functions this way. Not something that needs to happen in this PR though.

Reply:
Oh, this is perfect. Just yesterday I was asked if we had some *good first timer* issues in the code base. I'll make this one.